raven/src/intTest/java/org/opencadc/raven/FilesTest.java

```diff
@@ -296,16 +295,134 @@ public Object run() throws Exception {
         }
     }
 
-    public void checkHeadResult(HttpGet request, URI artifactURI, URI checksum, String contentType) {
+    @Test
+    public void testConsistencyHEAD() throws Exception {
```

Comment:
Since this method tests both HEAD and GET, it could be renamed, maybe testConsistencyPreventNotFound?